### DISCUSSION TITLE
Moving Kubescape helm chart to a separate repository

### DISCUSSION THREAD

**Discussion by yonahd** [kubescape/kubescape#299] [Ideas]
A couple months ago I contributed a helm chart to install kubescape [here](https://github.com/armosec/kubescape/tree/master/examples/helm_chart) as a cronjob. Possibly this should be moved to a seperate repo and helm tgz uploaded  so it can be installed like a helm chart should be installed e.g. https://github.com/armosec/armo-helm
(couldn't understand if the armo-helm has kubescape also included and if so a readme should be generated for it)

**Comment by slashben** [comment]
@yonahd , yes, we could move it to the other git repo indeed and it can be part of the armo *helm* repo as Kubescape (only) chart
